taskcluster/ac_taskgraph/transforms/build.py

```diff
@@ -74,6 +77,31 @@ def _get_timestamp(config):
     return push_date_time.strftime('%Y%m%d.%H%M%S-1')
 
 
+def _get_buildid(config):
+    push_date_string = config.params["moz_build_date"]
+    push_date_time = datetime.datetime.strptime(push_date_string, "%Y%m%d%H%M%S")
```

Comment:
Nit: `config.params.moz_build_date` is now available and it returns a regular datetime. This way, you don't have to manually parse the value anymore 😃

It was added in https://bugzilla.mozilla.org/show_bug.cgi?id=1587015

Reply:
Sweet, TIL! I've updated the function to use that instead, thanks!